test/verification.test.js

```diff
@@ -34,6 +34,14 @@ test("verifyRequestByKeyId()", async (t) => {
     return fetch(url, opts);
   }
 
+  const publicKeys = [
+    {
+      key: CURRENT_PUBLIC_KEY,
+      key_identifier: KEY_ID,
+      is_current: true,
+    },
+  ];
```

Comment:
I'd move it below `SIGNATURE` and use `PUBLIC_KEYS`, as the value does not change in-between tests.

Reply:
Done in https://github.com/copilot-extensions/preview-sdk.js/commit/482816832601ff055f5c43f3451a0928a617abc7 ✅